src/entities/card/organisms/card-preview.tsx

```diff
@@ -143,11 +120,21 @@ type ContentProps = Pick<Card, 'title' | 'content' | 'updatedAt'> &
   Pick<CardPreviewProps, 'href' | 'size'>;
 
 const Content = ({ content, title, href, size, updatedAt }: ContentProps) => {
+  const query = useSearchQuery();
+
   return (
     <ContentStyled>
       {/* FIXME: Add text-overflow processing */}
       <TextStyled type={TextType.header4}>
-        {href && <TitleLink to={href}>{title}</TitleLink>}
+        {href && (
+          <TitleLink to={href}>
+            {query !== '' ? (
+              <HighlightText query={query} entity={title} />
```

Comment:
[`nitpick:`](https://conventionalcomments.org/)

Мб так?

```suggestion
              <HighlightText query={query} text={title} />
```

Reply:
done